lib/build.js

```diff
@@ -341,10 +342,9 @@ const parseAct = (act_str, silent = false) => {
   return act;
 }
 
-
 const buildAct = config => ({act, oog, pass, name}) => {
-  console.log("build " + name)
-
+  console.log("building " + name)
```

Comment:
"build" plz

Reply:
cause "parse" is 5 letters as well :P

Reply:
Sounds like imperative form to me... Why is klab telling me to `build`? I think `building`, `parsing`, is a lot clearer.